oaknational/Oak-Web-Application
# Fix/portable text references

## Description
Ensure all pages receive resolved portable text references for use in "internal" links. Previously only blogs were hooked up to this, but now all slug/singleton methods are

## Issue(s)

Might fix, or at least progress #775 (Ensure all pages render internalLinks in portable text) - They should all now have the data to do it, any where that isn't rendering them is a UI bug

## How to test

1. Go to [the deploy](https://owa.thenational.academy/beta/webinars/getting-the-most-out-of-oaks-specialist-curriculum)
2. You should see a link to the lesson resource directory page under the "Further resources" heading

## Screenshots
Before (renders nothing, warns):
![CleanShot 2022-11-10 at 17 45 50](https://user-images.githubusercontent.com/2717635/201170591-0544e10f-315b-46ab-9a78-4d49224ada2c.png)

After:
![CleanShot 2022-11-10 at 17 53 45](https://user-images.githubusercontent.com/2717635/201170618-843be41c-c926-4aea-b3ff-e616617a8bd0.png)


## Checklist
